rhdm70-image-streams.yaml

```diff
@@ -0,0 +1,41 @@
+kind: List
+apiVersion: v1
+metadata:
+  name: rhdm70-image-streams
+  annotations:
+    description: ImageStream definitions for Red Hat Decision Manager 7.0
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: rhdm70-decisioncentral-openshift
+    annotations:
+      openshift.io/image.insecureRepository: 'true'
+      openshift.io/display-name: Red Hat Decision Manager Central 7.0
```

Comment:
Add the provider-display-name like this example: https://github.com/jboss-openshift/application-templates/blob/master/jboss-image-streams.json#L8

Reply:
It can contain this file, but not pointing to internal URLs, obviously. You can use public registry, even if this image does not exists there just yet. You can post process this file to make sure it pulls the image from correct registry.

Reply:
URLs changed to point to `registry.access.redhat.com`.